### DISCUSSION TITLE
Search field and button.

### DISCUSSION THREAD

**Discussion by Sandesh-Ghimire** [nirajgiriXD/garden-of-quotes#29] [General]
Can anyone design the search bar with buttons and provide search results in the body. currently its on the navbar itself.

**Comment by nirajgiriXD** [comment]
For search bar, the `quoteBox` component should be replaced by `seachBox` component, and the code should be modified [here](https://github.com/nirajgiriXD/garden-of-quotes/blob/main/src/App.tsx#L31).

```js
 isSearchBoxVisible ? <SearchBox /> : <QuoteBox quotes={quotes} />;
```

**Few points to consider:**
- The searchBox component should have similar css as the quoteBox
- In the searchBox, dropdown with suggestions should be present
- After user clicks on one of the suggestion, the search result should be displayed, the input field will also remain visible

**Comment by Sandesh-Ghimire** [comment]
Second and third points are already covered.
![Screenshot 2023-12-31 162905](https://github.com/nirajgiriXD/garden-of-quotes/assets/62834029/59dc5e8c-c00a-4e59-9aea-6bc75b8bc3b1)

but designing part is remaining.

**Reply by nirajgiriXD** [reply]
Looks great @Sandesh-Ghimire 🎉

Just a simple feedback: make the suggestions list's background color white and set the position absolute or implement any other mechanism to ensure that the header height is not being affected
